tests/system/feature_store/test_feature_store.py

```diff
@@ -2839,7 +2875,8 @@ def verify_purge(fset, targets):
         if target.name in target_names:
             driver = get_target_driver(target_spec=target, resource=fset)
             filesystem = driver._get_store().get_filesystem(False)
-            assert filesystem.exists(driver.get_target_path())
+            if filesystem is not None:
+                assert filesystem.exists(driver.get_target_path())
```

Comment:
So, this makes the test irrelevant to the new Redis driver (since it does not support returning a fs). What is the point of running this test on it, if it doesn't check anything for it?

Reply:
limited value - just excersizing the purge and verifying out-of-band that Redis deleted the keys.
I will check what can be with fsspec for Redis.

Reply:
No fsspec, as it's not really a file-system. However, some actions need to be supported - see my comment in the new review.

Reply:
added get(), put(), upload(), listdir()